man: fix typo and groff syntax in man pages

b3lix opened the pull request:
### Issues

- Fix typo of `DENTITY` to `IDENTITY`.
- Underline `FILE:...` using `\fI`, to match rest of the file.
- Incorrect groff syntax in man pages:
```
$ cd freeipa

$ groff -man -z client/man/ipa.1
troff: client/man/ipa.1:149: cannot use newline as a starting delimiter

$ groff -man -z client/man/epn.conf.5
troff: client/man/epn.conf.5:37: warning: numeric expression expected (got 'L')

$ groff -man -z client/man/default.conf.5
troff: client/man/default.conf.5:50: warning: numeric expression expected (got 'L')
troff: client/man/default.conf.5:270: warning: numeric expression expected (got 'L')
```

Resolves: https://pagure.io/freeipa/issue/9732

rcritten commented:
I'm not seeing incorrect rendering of the man pages. This is what it looks like to me.
current:
```
       --pkinit-identity=DENTITY
              Identity string for PKINIT authentication to use to join the IPA
              realm,  for  example  'FILE:/path/to/cert.pem,/path/to/key.pem'.
              See  krb5.conf(5)  for  more information. The option is mutually
              exclusive with --password and --keytab.
```
with patch (where FILE:... is underlined)
```
       --pkinit-identity=DENTITY
              Identity string for PKINIT authentication to use to join the IPA
              realm, for example FILE:/path/to/cert.pem,/path/to/key.pem.  See
              krb5.conf(5) for more information. The option is mutually exclu‐
              sive with --password and --keytab.
```

Note that there truly is a typo here. DENTITY should be IDENTITY.

The nroff docs define ' and . as control characters but only if they begin a line (by my brief reading anyway). So I'm not sure what the risk here is other than hygiene.

flo-renaud commented:
I agree with Rob's assessment, I don't think this patch is needed as the man pages already render properly. But you can use this PR to fix the issue `--pkinit-identity=DENTITY`

b3lix commented:
From the [PR](https://pagure.io/freeipa/issue/9665) mentioned I see changes that instead of using `'`  in 'FILE:...' or 'DIR:...', was using underlining it with `\fI`. Another thing when using bold `\fB`, then `'` around it was removed, not only on begging on the line. So my patch here was for consistency effort in this, to change other occurences.
Thanks for pointing out the `--pkinit-identity=DENTITY`, I can definitely use this PR to correct it.

b3lix commented:
I have updated the commit regarding discussion above and also introduced one commit fixing the incorrect groff syntax - updated info in PR description.

rcritten commented:
Since this includes the fix from https://github.com/freeipa/freeipa/pull/7682 should we close that PR? Should attribution be given?

Can you modify the subject of the original patch. It now only fixes some formatting/syntax issues.

b3lix commented:
Hi @rcritten ,
yes we can close PR [#7682](https://github.com/freeipa/freeipa/pull/7682). I have updated the last commit to give attribution, author is set to https://github.com/vasily-p, author of mentioned PR. Second commit message updated too.

menonsudhir commented:
master:

* a542a9185a127ac0202ac0c0b0bc255d11aaf355 man: fix formatting and syntax issues
* 071e5fe3764feba11edb664b718ad0e614b2ea66 man: fix incorrect groff syntax in man pages
